src/courses/beginner/08.md

```diff
@@ -11,8 +11,6 @@ headerDepth: 3
 ### Try it out!
 
 InSpec allows you to send your test results to one or more *reporters*, or output formats. The following command outputs results to both the terminal (`cli`) and to a JSON report file (`json:baseline_output.json`).
-<!-- 
-You can configure the reporter(s) using either the `--json-config` option or the `--reporter` option. While you can configure multiple reporters to write to different files, only one reporter can output to the screen (via stdout). -->
```

Comment:
json-config vs reporter is addressed later so we can drop it.  i think having a brief explanation in a details section explaining hey while your different reporter options can all write to different files, you're gonna run into issues if two or more of them try to write to the same file - and keep in mind that STDOUT is actually just a fancy "file"